pol.html

```diff
@@ -249,17 +250,17 @@ <h2 id="reference">Reference Standards</h2>
 		<p>The <abbr title="Worldwide Web Consortium">W3C</abbr> Web Accessibility Initiative (WAI) provides a set of standards relevant to various aspects of web accessibility. Referencing these standards in policy documents ensures clear criteria for accessibility that are commonly recognized by governments and organizations from around the world.</p>
 
 		<h3>Web Content Accessibility Guidelines (WCAG) 2.0</h3>
-		<p>Accessibility requirements for web content, including text, images, audio-visual materials, and the code used for structure, style, and interactions.</p>
-		<p>WCAG is applicable for any organization with a website, including mobile websites and applications.</p>
+		<p>Defines accessibility requirements for web content, including text, images, audio-visual materials, and the code used for structure, style, and interactions.</p>
+		<p>WCAG is applicable for any organization with a website, including websites with mobile content and applications.</p>
 		<p>Find out more at <a href="/WAI/intro/wcag">WCAG Overview</a>.</p>
 		
 		<h3>Authoring Tool Accessibility Guidelines (ATAG)</h3>
-		<p>Accessibility requirements for web authoring tools, which are software and services that are used to produce web content. Examples of authoring tools include HTML editors, content management systems (CMS), or online forums. ATAG guidelines outline both how to make the tools themselves accessible and also how the tools can be built to help create more accessible content.</p>
+		<p>Defines accessibility requirements for web authoring tools, which are software and services that are used to produce web content. Examples of authoring tools include HTML editors, content management systems (CMS), or online forums. ATAG guidelines outline both how to make the tools themselves accessible and also how the tools can be built to help create more accessible content.</p>
```

Comment:
Not sure if we had this in a previous iteration or not but it seems odd to just start with "accessibility requirements..." - it needs a linkage to the heading IMO.